i18n/ru/package.i18n.json

```diff
@@ -0,0 +1,76 @@
+  "espIdf.searchInEspIdfDocs.title": "ESP-IDF: Искать в документации...",
+  "espIdf.getEspAdf.title": "ESP-ADF: Установить ESP-ADF",
+  "espIdf.getEspMdf.title": "ESP-IDF: Установить ESP-MDF",
+  "debug.initConfig.name": "ESP-IDF: Запуск",
+  "debug.initConfig.description": "Новая конфигурация для запуска проектов ESP-IDF",
+  "param.adapterTargetName": "Целевое имя для адаптера отладки ESP-IDF",
+  "param.openOcdConfigFilesList": "Список файлов конфигурации в каталоге OpenOCD Scripts",
+  "param.port": "Путь к выбранному порту устройства",
+  "param.flashBaudRate": "Скорость передачи данных флеш-памяти ESP-IDF",
+  "param.pythonBinPath": "Абсолютный двоичный путь Python, используемый для выполнения скриптов Python ESP-IDF",
+  "param.espIdfPath": "Путь для поиска платформы ESP-IDF (IDF_PATH)",
+  "param.espAdfPath": "Путь для поиска платформы ESP-ADF (ADF_PATH)",
+  "param.espMdfPath": "Путь для поиска платформы ESP-MDF (MDF_PATH)",
+  "param.toolsPath": "Путь для поиска инструментов ESP-IDF (IDF_TOOLS_PATH)",
+  "param.exportPaths": "Пути, добавляемые к PATH",
+  "param.exportVars": "Переменные, которые нужно добавить к системным переменным среды",
+  "param.useIDFKConfigStyle": "Включить/отключить проверки стиля ESP-IDF для файлов Kconfig",
+  "param.showOnboardingOnInit": "Показать окно конфигурации ESP-IDF",
+  "param.coveredLightTheme": "Цвет фона для охватываемых линий в светлой теме для покрытия ESP-IDF.",
+  "param.coveredDarkTheme": "Цвет фона для охватываемых линий в темной теме для покрытия ESP-IDF.",
+  "param.partialLightTheme": "Цвет фона для частично покрытых линий в светлой теме для покрытия ESP-IDF.",
+  "param.partialDarkTheme": "Цвет фона для частично покрытых линий в темной теме для покрытия ESP-IDF.",
+  "param.uncoveredLightTheme": "Цвет фона для непокрытых линий в светлой теме для покрытия ESP-IDF.",
+  "param.uncoveredDarkTheme": "Цвет фона для непокрытых линий в темной теме для покрытия ESP-IDF.",
+  "view.components.name": "Компоненты проекта",
+  "configuration.title": "ESP-IDF",
+  "espIdf.apptrace.archive.refresh.title": "Обновить список архива трассировки",
+  "espIdf.apptrace.archive.showReport.title": "Показать отчет о трассировке",
+  "espIdf.apptrace.customize.title": "Настроить параметры для трассировки приложения",
+  "trace.poll_period.description": "poll_period будет установлен для трассировки приложения",
+  "trace.trace_size.description": "trace_size будет установлен для трассировки приложения",
+  "trace.stop_tmo.description": "stop_tmoбудет установлен для трассировки приложения",
+  "trace.wait4halt.description": "wait4halt будет установлен для трассировки приложения",
+  "trace.skip_size.description": "skip_size будет установлен для трассировки приложения",
+  "param.saveBeforeBuildDescription": "Сохраните все отредактированные файлы в рабочей области, прежде чем продолжить сборку, хотя, если не удается сохранить файлы, она все равно будет построена",
```

Comment:
@brianignacio5  Should we improve the English version of the sentence? It's unclear.